### PR TITLE
kubeadm/phases: small grammar improvements

### PR DESCRIPTION
**Pull request by neolit123**
**What this PR does / why we need it**:
small grammar fixes and cleanup in `kubeadm/phases`.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
none

**Release note**:

```release-note
NONE
```
Lubomir (VMware)

### CODE REVIEW

**Comment by neolit123** [comment]
@kubernetes/sig-cluster-lifecycle

**Comment by neolit123** [comment]
@kubernetes/sig-cluster-lifecycle-misc

**Comment by luxas** [comment]
/ok-to-test

assigned @fabriziopandini as he has synced with sig-docs on the naming conventions

**Comment by fabriziopandini** [comment]
@neolit123, thanks for your PR 👍 

Starting from 1.9 cobra commands documentation for kubeadm is used also in the kubernetes web site, so we are tied to the documentation [style guide](https://kubernetes.io/docs/home/contribute/style-guide/).

With regard to your PR, I guess that the most relevant is "Use camel case for API objects" that defines where special capitalisation rules should be applied; kubeadm or kubeconfig are not api objects, so we have to use small caps. Please fix this.

On top of that, during 1.9 cycle there was an effort to standardise help messages with other kubernetes commands like kubectl, so we removed all the dots at the end of commands short description and at the of of command flag description (NB. unfortunately is still not applied in 100% cases, but this seems to be the convention there). Please fix this as well.

**Comment by neolit123** [comment]
thank you for the information, @fabriziopandini.
i will try to find time to amend the patch during the holidays, but probably this will happen after new year.

**Comment by neolit123** [comment]
@fabriziopandini,

HNY! commit updated.

**Comment by fabriziopandini** [comment]
/lgtm
Thanks @neolit123 !

**Review by fabriziopandini — APPROVED** [review]

File: cmd/kubeadm/app/cmd/phases/addons.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func NewCmdAddon() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "addon",
 		Aliases: []string{"addons"},
-		Short:   "Installs required addons for passing Conformance tests",
+		Short:   "Installs required addons for passing conformance tests",
```
Comment (timothysc) [inline cmd/kubeadm/app/cmd/phases/addons.go]: Given the name throughout the code is [Conformance] the capitalization is consistent.

File: cmd/kubeadm/app/cmd/phases/bootstraptoken.go [hunk 2]
```diff
@@ -97,7 +97,7 @@ func NewCmdBootstrapToken() *cobra.Command {
 		Aliases: []string{"bootstraptoken"},
 	}
 
-	cmd.PersistentFlags().StringVar(&kubeConfigFile, "kubeconfig", "/etc/kubernetes/admin.conf", "The KubeConfig file to use when talking to the cluster")
+	cmd.PersistentFlags().StringVar(&kubeConfigFile, "kubeconfig", "/etc/kubernetes/admin.conf", "The kubeconfig file to use when talking to the cluster")
```
Comment (timothysc) [inline cmd/kubeadm/app/cmd/phases/bootstraptoken.go]: same with KubeConfig.

**Review by timothysc — CHANGES_REQUESTED** [review]
Minor comments about consistency with the rest of the codebase, otherwise lgtm.

File: cmd/kubeadm/app/cmd/phases/bootstraptoken.go (hunk 2 above)
Comment (neolit123) [reply cmd/kubeadm/app/cmd/phases/bootstraptoken.go]: @timothysc, @fabriziopandini mentioned above that `kubeconfig` should be lower case:

> kubeadm or kubeconfig are not api objects, so we have to use small caps.

maybe the same applies to `Conformance` i have no information if that's an API object?

Comment (timothysc) [reply cmd/kubeadm/app/cmd/phases/bootstraptoken.go]: It's a keyword reference in the e2e tests, and it's capitalized.  I'd verify by looking through the docs and being consistent there.

Comment (neolit123) [reply cmd/kubeadm/app/cmd/phases/bootstraptoken.go]: odd, there seems to be a small contradiction between @fabriziopandini and @timothysc .
i will make the changes as @timothysc requests.

**Comment by neolit123** [comment]
@timothysc @fabriziopandini updated.

**Comment by neolit123** [comment]
rebased the PR against the latest master to see if the CI error will be solved.

**Comment by neolit123** [comment]
i have no solutions for the above failing CI errors. please, advice how to proceed.

**Comment by neolit123** [comment]
`pull-kubernetes-unit` seems to spill a pile of errors that i don't understand. could someone please explain these?

**Review by timothysc — APPROVED** [review]
/approve

**Review by timothysc — APPROVED** [review]
/lgtm 

/retest all

**Pull request merged**